mirage/config.js

```diff
@@ -516,6 +516,16 @@ export default function () {
     }
   });
 
+  this.get('/owner/:provider/:login/allowance', function (schema, request) {
+    let owner = schema.users.where({ login: request.params.login }).models[0];
```

Comment:
This won't work for organizations. Is it expected?

Reply:
right, I extended it for looking for orgs as well.